collaboration.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Collaboration Tools
+layout: topic
+categories: guide
+---
+A key component of a successful hackathon is being able to efficiently 
+collaborate with other members of your team. There are many different 
+free tools that can help you with this. Some collaboration tools that 
+might support your project include but are not limited to:
+
+## Code Collaboration Tools
```

Comment:
Change to Version Control instead of Code Collaboration? Or some combination thereof?